doc/documentation.rst

```diff
@@ -24,6 +24,6 @@ typically with `scikit-learn <http://scikit-learn.org>`_ with its
    assembling
    encoding
    cleaning
+   datasets
```

Comment:
I don't really like that we are dedicating a top-level header to something which is actually a fairly minor feature of skrub.

I think that it is fine for now, but in the long run, as we add other somewhat "misc" sections, we should fold this section into a subsection.